src/components/side_panel/chart/building_blocks/series_design/series_with_axis_design_editor.ts

```diff
@@ -112,9 +112,14 @@ export class SeriesWithAxisDesignEditor extends Component<Props, SpreadsheetChil
         break;
       case "exponential":
       case "logarithmic":
-      case "trailingMovingAverage":
         config = { type };
         break;
+      case "trailingMovingAverage":
+        config = {
+          type,
+          window: this.defaultWindowSize,
```

Comment:
This seems not needed ? The trailing average trendline is typed to have the window being optional. And I don't think this should have anything to do with a commit handling the xlsx export/import

Reply:
Yeah it's now on a separate commit, but it doens't really address the issue. window is still typed as possibly undefined in a trendline. Either 1) it should be required, typed as such and we should do a migration to ensure it's always defined or 2) your code should also work when window is undefined, and defaults to `DEFAULT_WINDOW_SIZE` size. 

2) sounds way better

Reply:
not sure of what this commit does. Is it a UI fix?  if so it probably does not belong to this PR but the content seems deeply intertwinded with excel export / import so I don't really udnerstand what it should do. Is the split between the commits correctly done?

Reply:
Then I think it's better not to make this modification.
If I understand correctly, the window is currently not required and the code can work as it is.  If you change the entry point that allowed the window to be undefined, we risk a future situation  where we will not understand how a user was able to generate a trendline with a window undefined since the code will show that it's not possible. Going back to the commit history to find this specific modification could take a lot of time for nothing.

I think that this modification should come in place if we decide that having trend.window undefined was a mistake and that we need to make a migration to make things right.